translation-web/src/main/java/eu/europeana/api/translation/config/WebMvcConfig.java

```diff
@@ -81,5 +87,18 @@ private Map<String, MediaType> getMediaTypesMapping() {
 //  @Override
 //  public void addResourceHandlers(ResourceHandlerRegistry registry) {
 //    registry.addResourceHandler("/public/**").addResourceLocations("classpath:/public/");
-//  }  
+//  }
+
+
+
+
+  private ObjectMapper objectMapper = new ObjectMapper();
+
+  @Autowired
+  private ResourcePatternResolver resourcePatternResolver;
+
+ @Override
+ public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers) {
```

Comment:
please add a comment why are we doing this. something like this - 

  /**
   * we need to tell Spring about our JsonSchemaValidatingArgumentResolver.
   * We do this by using the addArgumentResolvers(..) method from the WebMvcConfigurer interface.
   * @param resolvers
   */